tools/tpkg/main.go

```diff
@@ -22,27 +22,44 @@ import (
 	"strings"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 	"github.com/toitlang/tpkg/commands"
 	"github.com/toitlang/tpkg/config"
 	"github.com/toitlang/tpkg/config/store"
 	"github.com/toitlang/tpkg/pkg/tracking"
 )
 
-var (
-	rootCmd = &cobra.Command{
-		Use:              "toitpkg",
-		Short:            "Run pkg commands",
-		TraverseChildren: true,
-	}
-)
-
 func getTrimmedEnv(key string) string {
 	return strings.TrimSpace(os.Getenv(key))
 }
 
 // Should be set with '-X' flag when linking.
 var sdkVersion = ""
 
+func copyAsDeprecated(cmd *cobra.Command) *cobra.Command {
```

Comment:
Does this mean anything for 'jag pkg'?

Reply:
No.
Jag uses the tpkg library the same way we do here.
Only difference is that Jag uses the pkgCommand as a subcommand and not as root command.